sdks/java/extensions/sql/build.gradle

```diff
@@ -29,33 +29,8 @@ applyJavaNature(
   enableStrictDependencies: true,
   shadowClosure: {
     dependencies {
-      include(dependency(library.java.guava))
-      include(dependency(library.java.protobuf_java))
-      include(dependency(library.java.protobuf_java_util))
-      include(dependency("org.apache.calcite:.*"))
-      include(dependency("org.apache.calcite.avatica:.*"))
-      include(dependency("org.codehaus.janino:.*"))
+      include(dependency(library.java.vendored_calcite_1_19_0))
```

Comment:
You wouldn't need to shade this dependency at all and could disable shading within this gradle file.